routes/web.php

```diff
@@ -20,9 +18,17 @@
 | contains the "web" middleware group. Now create something great!
 |
 */
-
 Route::get('/',[LoginController::class,'create']);
-Route::post('/login',[LoginController::class,'store'])->middleware('loginvalidate');
+Route::post('/login',[LoginController::class,'login']);
+Route::get('/adduserform',[RegisterController::class,'create']);
+Route::post('/adduser',[RegisterController::class,'store']);
+Route::get('/adminform',[PostController::class,'admincreate']);
+Route::get('/userform',[PostController::class,'usercreate']);
+Route::get('/adminhome',[PostController::class,'adminindex']);
+Route::get('/userhome',[PostController::class,'userindex']);
+Route::get('/logout',[LogoutController::class,'logout']);
+Route::post('/adminpost',[PostController::class,'store']);
+Route::post('/userpost',[PostController::class,'store']);
```

Comment:
is there any reason we need seperate route for each. i think this can be done smthing like this

`Route::post('/posts',[PostController::class,'store']);`